src/test/common/steps/overview-steps.js

```diff
@@ -17,3 +18,24 @@ Then(/^I am shown the overview page$/, async function () {
 Then(/^I am shown the first page of the application$/, async function () {
   await pages.waitForFirstPage()
 })
+
+// The original session id is stored on the Overview page, but to get a new session id we need to make sure
+// that we navigate to a subsequent page (back to the Overview page is sufficient) to check for a new session id
+// We also verify that the lang cookie has been removed
+Then(/^my session has been destroyed$/, async function () {
+  const pageSessionId = await pages.overview.sessionId
```

Comment:
does this need an await - is getting overview.sessionId asynchronous? I expect it works iwth the await anyway...

Reply:
That's true, it doesn't need it, it was there because that was originally retrieved using an asynchronous method, I'll correct it.